servicedef/package_info.go

```diff
@@ -0,0 +1,6 @@
+// Package servicedef contains definitions for the REST protocol that test services must
+// implement. See the top-level README.md for more details.
+//
+// The package is used by the test harness, but can also be imported by any test service
+// code that is Go-based.
+package servicedef
```

Comment:
These types used to be in the `ssetests` package— I just moved and renamed them a bit to make things look a bit more like the equivalent `sdk-test-harness` code, which I found to be clearer.